packages/be-gateway/src/services/storage.service.ts

```diff
@@ -115,6 +115,11 @@ export class StorageService {
     const totalSize = await storageCache.getTotalSize()
     const { maxStorageSize } = await mdOrgGetOne(organizationId)
 
+    //  unlimited storage size
+    if (maxStorageSize === -1) {
+      return true
+    }
+
     if (maxStorageSize && totalSize > maxStorageSize) {
```

Comment:
This line is not related to the PR but if a reviewer I will think it is checking `maxStorageSize` is not `null` or `undefined`. How about the variable `maxStoreageSize` can be 0?

Reply:
@binhtran432k thank for your comment
Actually, this PR still under development.
I often create the PR in development process to self review my updates